### PR TITLE
include path and x_ms_path endpoints in genearted crates

### PR DESCRIPTION
**Pull request by bmc-msft**
This addresses an issue where some specs split endpoints between `path`
and `x-ms-path`.  This exposes a significant number of operations across
the generated crates that were not previously included.

NOTE: this was generated using version of the REST specs, 48d85585897aa6ed448ca689b094e60377a25cb7

### CODE REVIEW

**Comment by bmc-msft** [comment]
Note, this PR (or something that accomplishes the same goal) is required to use the generate crates as a backend to the newly added `azure_device_update` crate.

**Comment by bmc-msft** [comment]
The failures identified in the build are not part of this PR and will be addressed in #719

File: services/mgmt/kusto/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ The default tag is `package-2022-02`.
 
 The following [tags](https://github.com/Azure/azure-sdk-for-rust/blob/main/services/tags.md) are available:
 
-- `package-2022-02` has 1 operations from 1 API versions: `2022-02-01`. Use crate feature `package-2022-02` to enable. The operations will be in the `package_2022_02` module.
+- `package-2022-02` has 70 operations from 1 API versions: `2022-02-01`. Use crate feature `package-2022-02` to enable. The operations will be in the `package_2022_02` module.
```
Comment (cataggar) [inline services/mgmt/kusto/README.md:13]: yay 🚀

File: services/autorust/openapi/src/openapi.rs [hunk 2]
```diff
@@ -54,12 +54,17 @@ pub struct OpenAPI {
...
+        let mut result = IndexMap::new();
+        for (k, v) in self.x_ms_paths.iter() {
+            result.insert(k.clone(), v.clone());
         }
+        for (k, v) in self.paths.iter() {
+            if !self.x_ms_paths.contains_key(k) {
+                result.insert(k.clone(), v.clone());
+            }
+        }
+        result
```
Comment (cataggar) [inline services/autorust/openapi/src/openapi.rs:67]: Looks good

**Review by cataggar — APPROVED** [review]

**Pull request merged**